docs/migration.md

```diff
@@ -1,235 +1,288 @@
-can't migrate at the same time, it must be made sure that the server is able
-to support multiple versions of rdiff-backup at the same time.
-
-> **NOTE:** the same approach can be used to support multiple clients of
-	different versions but the use case doesn't seem as useful, hence
-	it is left to the interpretation of the reader.
-
-### Server side
-
-Python [virtual environments](https://docs.python.org/3/glossary.html#term-virtual-environment)
-are a mean to create different installations of Python
-libraries, without risk of conflicting libraries, exactly what we need for
-our purpose.
-
-You can use `venv` or `virtualenv` to create virtual environments, it's rather
-a matter of taste with Python 3. With Python 2, you might want to stick to
-`virtualenv`. In the following lines we'll use `virtualenv` and shorten virtual
-environments into "virtualenvs".
-
-For each version which you want to install, create the virtualenvs,
-install rdiff-backup in them, then verify it's properly installed
-(here with rdiff-backup 2.0 as example):
-
-```
-virtualenv ${BASEDIR}/rdiff-backup-2.0
-${BASEDIR}/rdiff-backup-2.0/bin/pip install rdiff-backup==2.0.5
-${BASEDIR}/rdiff-backup-2.0/bin/pip install pylibacl pyxattr  # optional
-${BASEDIR}/rdiff-backup-2.0/bin/rdiff-backup --version  # result is 2.0.5
-```
-
-> **NOTE:** you can also only create major versions virtualenvs, like
-	`rdiff-backup-2`, or even z-Versions like `rdiff-backup-2.0.5` but
-	the middle version seems like a reasonable middle-way.
-
-Optionally, you can add to your PATH an executable script `rdiff-backup-2.0`
-with a content like the following, so that the clients don't need to care
-about the full-path (which will be our assumption in the following lines):
-
-```
-#!/bin/sh
-BASEDIR=/usr/local/lib
-exec ${BASEDIR}/$(basename $0)/bin/rdiff-backup "$@"
-```
-
-> **TIP:** the `basename` trick allows you to only maintain one script,
-	linked (hard or soft) under multiple names.
-
-Repeat for each version of rdiff-backup you want to maintain in parallel.
-
-
-### Client side
-
-The client side is even simpler, you just need to use **\--remote-schema**
-pointing at the correct version of rdiff-backup, something like:
-
-```
-rdiff-backup --remote-schema 'ssh -C %s rdiff-backup-2.0 --server' \
-	-b /sourcedir user@serverhost::/backup-repo
-```
-
-Starting with rdiff-backup 2.1+, the command would look like this and
-wouldn't need to be changed with each update of the client, as the version
-would automatically follow:
-
-```
-rdiff-backup --remote-schema 'ssh -C {h} rdiff-backup-{vx}.{vy} server' \
-	backup /sourcedir user@serverhost::/backup-repo
-```
-
-> **TIP:** for older versions of rdiff-backup, one could surely write a
-	wrapper script mimicking the same behaviour, using something along the
-	line of `$(rdiff-backup --version | awk -F'[. ]' '{print $2 "." $3}')`.
-
-And that's it for the side-by-side installation...
+# Migration and Side-by-Side installation
```

Comment:
Is it a line ending issue ??

Reply:
Yes, as @deece noted, I should have split the commits (sorry!), but I'm happy that @bigpresh brought the trick with ignoring the blanks. I didn't notice before, but the option is available from the UI: 
![image](https://user-images.githubusercontent.com/3105685/110367607-2d3e7280-8048-11eb-845c-9ed93c24c5e3.png)